com.meetfranz.Franz.appdata.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.meetfranz.Franz.desktop</id>
+  <name>Franz</name>
+  <project_license>Apache-2.0</project_license>
+  <developer_name>Stefan Maizner</developer_name>
+  <summary>Messenger for the desktop</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://meetfranz.com/</url>
```

Comment:
You could add other url's like bugtracker, help, faq or donation

Reply:
instead of that homepage? or just multiple `<url/>`?

Reply:
multiple url's